packages/cmrjs/cmr-utils.js

```diff
@@ -300,6 +300,13 @@ function constructOnlineAccessUrls(files, distEndpoint, buckets) {
     }
   });
 
+  urls.push({
+    URL: urljoin(distEndpoint, s3CredsEndpoint),
+    URLDescription: 'S3 Credentials Endpoint',
+    Description: 'S3 Credentials Endpoint',
+    Type: 'GET DATA'
+  });
```

Comment:
The idea here was simply to add the `s3CredsEndpoint` to the list of onlineAccessUrls at the Granule level.

Reply:
Is this the right type of `Type` for this link?  I don't know the answer, I didn't look closely 
could `USE SERVICE API` be better?

Possible types are 
```
      "enum": ["DOWNLOAD SOFTWARE", "EXTENDED METADATA", "GET DATA", "GET RELATED VISUALIZATION", "GOTO WEB TOOL", "PROJECT HOME PAGE", "USE SERVICE API", "VIEW RELATED INFORMATION"]
```
 /shrug?

Reply:
shrug.... i mean it sounds good. Could ask erich reiter for clarification.

Reply:
I don't have a preference. I wasn't able to find documentation on the intent of each of those types online. I'll toss a message to Erich.

Reply:
I spoke with CMR folk (and Erich) - their verdict is to use `VIEW RELATED INFORMATION` as the tag.